lint/analyzers_test.go

```diff
@@ -42,6 +44,16 @@ func testAnalyzers(t *testing.T, code string, analyzers ...*analysis.Analyzer) [
 		nil,
 	)
 
+	// Suppress parser errors
+	config.HandleParserError = func(err analysis.ParsingCheckingError, program *ast.Program) error {
+		return nil
+	}
+
+	// Suppress checker errors
+	config.HandleCheckerError = func(err analysis.ParsingCheckingError, checker *sema.Checker) error {
+		return nil
+	}
```

Comment:
Why are the parsing and checking errors suppressed for all tests that use the utility?

This seems brittle, as it's likely that a test case might accidentally have a syntactical or semantic problem during development of the test case, and resulting errors being dropped will likely be very confusing.

Maybe make this optional and only explicitly pass these error handlers in the test cases where they are needed.